server/mlabns/tests/test_resolver.py

```diff
@@ -423,21 +468,32 @@ def testAnswerQueryWhenMoreThanFourToolsAreEquallyClosest(self):
         query.longitude = 0.0
 
         mock_fetched_tools = [
-            _createSliverTool(
-                _TOOL_ID, site_id='abc01', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='abc02', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='abc03', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='abc04', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='abc05', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='abc06', latitude=1.0, longitude=1.0),
-            _createSliverTool(
-                _TOOL_ID, site_id='cba01', latitude=5.0, longitude=5.0)
-            ]
+            _createSliverTool(_TOOL_ID,
+                              site_id='abc01',
+                              latitude=1.0,
+                              longitude=1.0), _createSliverTool(_TOOL_ID,
+                                                                site_id='abc02',
+                                                                latitude=1.0,
+                                                                longitude=1.0),
+            _createSliverTool(_TOOL_ID,
+                              site_id='abc03',
+                              latitude=1.0,
+                              longitude=1.0), _createSliverTool(_TOOL_ID,
+                                                                site_id='abc04',
+                                                                latitude=1.0,
+                                                                longitude=1.0),
+            _createSliverTool(_TOOL_ID,
+                              site_id='abc05',
+                              latitude=1.0,
+                              longitude=1.0), _createSliverTool(_TOOL_ID,
+                                                                site_id='abc06',
+                                                                latitude=1.0,
+                                                                longitude=1.0),
+            _createSliverTool(_TOOL_ID,
+                              site_id='cba01',
+                              latitude=5.0,
+                              longitude=5.0)
+        ]
```

Comment:
I don't think it's this function in particular, but I notice YAPF gets weird when there's Python lists of function calls.

We want to disable YAPF any time it formats the code in a way that looks bizarre. This to me looks pretty weird and not really like what a human would write.